src/eos/chemTab.cpp

```diff
@@ -153,11 +150,49 @@ void ablate::eos::ChemTab::LoadBasisVectors(std::istream &inputStream, std::size
 #define safe_free(ptr) \
     if (ptr != NULL) free(ptr)
 
-void ablate::eos::ChemTab::ChemTabModelComputeFunction(PetscReal density, const PetscReal densityProgressVariable[], PetscReal *predictedSourceEnergy, PetscReal *progressVariableSource,
+void ablate::eos::ChemTab::ChemTabModelComputeFunction(PetscReal density, const PetscReal densityProgressVariables[], PetscReal *densityEnergySource, PetscReal *densityProgressVariableSource,
                                                        PetscReal *densityMassFractions) const {
+    ChemTabModelComputeFunction(&density, &densityProgressVariables, &densityEnergySource, &densityProgressVariableSource, &densityMassFractions, 1);
+}
+
+// Verified to work 11/7/23
+// NOTE: id arg is to index a potentially batched outputValues argument
+void ablate::eos::ChemTab::extractModelOutputsAtPoint(const PetscReal density, PetscReal *densityEnergySource, PetscReal *densityProgressVariableSource, PetscReal *densityMassFractions,
```

Comment:
Would you mind changing the names to match the google style case? https://ablate.dev/content/development/Formatting.html